### ISSUE TITLE
New Menu item has 1.5ish intermediary step

### ISSUE THREAD

**Issue by elinw** [Joomla3-Admin-template/joomla-cms#31]
When creating a new menu item there is a screen presenting a list of options, kind of like in 1.5. But then even if you select an option it disappears.

**Comment by kyleledbetter** [comment]
Yup, trying to populate the edit view with the selected choice. I changed the first step to be the item type, but I'm having trouble populating the edit view since the javascript is all crazy

**Comment by elinw** [comment]
THat is something Louis and Andrew worked extensively on for 1.6 and then
it was changed at some point along to way to something completely new
because someone had some objection or other  (around 1.7 I think). You
might want to look back at Louis's original work and work from there.

On Tue, Jul 24, 2012 at 12:58 PM, Kyle Ledbetter <
reply@reply.github.com

**Comment by kyleledbetter** [comment]
I had both those guys look at it this week and they both acted like it'd be a lot of work to change :P One of our dev volunteers is taking a last stab at it, might not get this item in alpha2

**Comment by elinw** [comment]
I'd really at least as a first step put it back to what it was at 1.6. If
you want I can test what happens if you do that.

Elin

On Wed, Jul 25, 2012 at 7:36 PM, Kyle Ledbetter <
reply@reply.github.com
